### PR TITLE
doc: remove todos, add note on URLs

### PR DESCRIPTION
**Pull request by aatifsyed**
Now tracked here: https://github.com/ChainSafe/filecoin-common-node-api/issues

### CODE REVIEW

File: FIPS/fip-common_node_api.md [hunk 1]
```diff
@@ -49,6 +49,9 @@ Additionally, by establishing a specification we seek to establish a greater res
 Using the OpenRPC specification format we define several groups of methods. These methods are defined in the OpenRPC specification found here: https://github.com/ChainSafe/filecoin-openrpc-spec. A list of the methods is included below for posterity.
 
 All methods are reproduced identically to their present implementation in Lotus 1.26.1.
+Note that Lotus hosts two different API's at HTTP paths [`/rpc/v0`](https://github.com/filecoin-project/lotus/blob/v1.28.0-rc5/documentation/en/api-v0-methods.md) and [`/rpc/v1`](https://github.com/filecoin-project/lotus/blob/v1.28.0-rc5/documentation/en/api-v1-unstable-methods.md),
+with the latter being marked as _unstable_.
+This document aims to specify the _latest_ API version (in this case, `v1`) and breaking changes will be negotiated through further FIPs.
```
Comment (ansermino) [inline FIPS/fip-common_node_api.md]:
```suggestion
This document aims to specify the `V1` API.
```

"latest" and "version" are a bit confusing here as the version of Lotus is mentioned just before. Additional changes via FIP is defined below.

File: FIPS/fip-common_node_api.md [hunk 2]
```diff
@@ -330,19 +333,5 @@ A better API, one that is well defined and carefully constructed, will ease the
...
-- [ ] Publish two versions of the document, one per URI
-- [x] Make ChainNotify definition more generic
-- [ ] Add link to playground
-- [ ] Perform proper spellcheck
-- [ ] Determine if this should be FIP or FRC
-- [ ] Setup discussion forum and add to header
-- [ ] Reach out to Venus to establish the state of their RPC
-- [ ] Transfer spec repo to filecoin-project and update links above
-- [ ] Add examples to all methods in spec
-
```
Comment (ansermino) [inline FIPS/fip-common_node_api.md]: Can't leave a suggestion here, but would suggest an updated list that focuses on what is relevant to readers :
```
- [ ] Improve descriptions for included methods
- [ ] Transfer spec repo to filecoin-project and update links above
- [ ] Add examples to all methods in spec
```

Lmk if you disagree with any of these items.

**Review by ansermino — CHANGES_REQUESTED** [review]
For the v0/v1 APIs - I recall we were going to define the path that the API should be served one, whereas these changes seem to only specify which part of the Lotus API we are referencing. Am I remembering that correctly?

**Comment by aatifsyed** [comment]
> For the v0/v1 APIs - I recall we were going to define the path that the API should be served one, whereas these changes seem to only specify which part of the Lotus API we are referencing. Am I remembering that correctly?

Yes, we floated the idea of e.g a `rpc/common-node-api/v1` HTTP path early on.
But we then decided to be "purely descriptive", deferring the following:
- URL paths
- API versioning

I've written this section from that descriptivist PoV - does that match your expectation?

**Review by ansermino — APPROVED** [review]

**Pull request merged**